codemark/secrets.py

```diff
@@ -6,17 +6,17 @@
 SERVICEACCOUNTFILE = os.path.join(dir_path,"firebase", "res", "service-account-file.json")
 api_key= os.getenv("OPENAI_API_KEY")# OPENAI KEY
 dbURL = os.getenv("FIREBASE_DATABASE_URL")
+FIREBASE_STORAGE_BUCKET_URL = os.getenv("FIREBASE_STORAGE_BUCKET_URL")
```

Comment:
Consider renaming the variable to STORAGE_BUCKET_URL, as it might break in the API secret vars in code submissions presumably